llvm/docs/CommandGuide/llvm-cxxfilt.rst

```diff
@@ -52,6 +52,11 @@ OPTIONS
 
   Do not demangle function parameters or return types.
 
+.. option:: --quote
```

Comment:
Nit: This should appear after `--no-strip-underscore` to maintain alphabetical order.

Reply:
That makes sense.